app/views/home/_project-group.html.erb

```diff
@@ -1,13 +1,19 @@
-<div>
-  <div>
-    <div><%= group[:icon] %></div>
-    <div><%= group[:title] %></div>
-    <div><%= group[:body] %></div>
-    <div>See all <%= group[:projects_count] %> projects</div>
+<div class="mb-8">
+  <div class="flex mb-4 px-4">
+    <div class="w-16 content-center"><%= inline_svg_pack_tag "media/svgs/group-#{group[:name].downcase}.svg" %></div>
+    <div class="w-full">
+      <div><%= section_name group[:name] %></div>
+      <div class="w-full flex flex-col sm:flex-row justify-between flex-shrink-0">
+        <div class="flex-1">
+          <%= group[:body] %>
+        </div>
+        <%= link_to projects_path(project_types: group[:project_types].join(',')), class: 'float-right text-indigo-700 font-bold group' do %>
+          See all <%= group[:projects_count] %> projects <%= inline_svg_pack_tag 'media/svgs/chevron-right.svg', class: 'inline-block fill-current transform transition-transform duration-100 -translate-x-1 group-hover:translate-x-0' %>
+        <% end %>
+      </div>
+    </div>
   </div>
-  <div>
-    <% group[:featured_projects].take(3).each do |project| %>
-      <%= render partial: 'projects/home-project', locals: {project: project} %>
-    <% end %>
+  <div class="w-full px-4 sm:px-0 space-y-bottom-4 sm:grid grid-cols-2 lg:grid-cols-3 sm:gap-6">
+    <%= render partial: 'projects/project-card', collection: group[:featured_projects], locals: { group: group[:name] }, as: :project, cached: true %>
```

Comment:
Passing the group name as a local... but with caching it seems to not actually work well? Does it cache the whole view or each individual card?

Reply:
It's each individual card that's being cached.

Reply:
We don't need to set it a local afaik. `as: :project` should do that for us from the `collection` attribute.
What is the actual issue with the caching?

Reply:
regarding https://github.com/helpwithcovid/covid-volunteers/pull/113/commits/76e40cf6ac8b8b37f10bc7bfc2f0ed1827b9f16c - i did this because projects will exist in multiple types.. meaning they could belong in more than one group. I think the design goal was to make each row a unique color that matches the group (so that every project in a row had the same color). If you turn caching off, all the colors match. 

 It's super minor, but i thought it was a nice bit a polish worth trying.

Reply:
I see what you mean about the colors. Nice touch!